src/pages/[platform]/build-a-backend/auth/concepts/tokens-and-credentials/index.mdx

```diff
@@ -258,7 +258,6 @@ Token revocation is enabled automatically in Amplify Auth. To revoke tokens you
 
 ## Next steps
 
-- [Learn how to customize the ID token](/[platform]/build-a-backend/auth/examples/customize-id-token/)
-- [Learn how to customize the access token](/[platform]/build-a-backend/auth/examples/customize-access-token/)
```

Comment:
Is there no new link for customizing the access token?

Reply:
@aspittel advised to remove this link